### PR TITLE
Improve the description of difflib in the documentation

### PR DESCRIPTION
**Pull request by Mandeepahlawat**
From "can produce difference information in various formats ..."
to " can produce information about file differences in various formats ..."

Automerge-Triggered-By: @Mariatta

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

## Recognized GitHub username

We couldn't find a [bugs.python.org](https://bugs.python.org/) (b.p.o) account corresponding to the following GitHub usernames:

@Mandeepahlawat

This might be simply due to a missing "GitHub Name" entry in one's b.p.o account settings. This is necessary for legal reasons before we can look at this contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.




You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

**Review by Mariatta — DISMISSED** [review]
Thanks!

**Review by Mariatta — COMMENTED** [review]
Hmm reading the entire paragraph again, I think "difference" is correct here.

**Comment by Mandeepahlawat** [comment]
> Hmm reading the entire paragraph again, I think "difference" is correct here.

@Mariatta yeah, i just read it again and in this context "difference" would be correct, but i don't think the whole sentence is grammatically correct. Although, "diff" would definitely be better

**Comment by vsajip** [comment]
> but i don't think the whole sentence is grammatically correct. 

I can't see anything grammatically wrong with that sentence. However, "difference information" could be replaced by "information about differences" to avoid someone revisiting this because it looked wrong at first sight.

File: Doc/library/difflib.rst [hunk 1]
```diff
@@ -18,7 +18,7 @@
 --------------
 
 This module provides classes and functions for comparing sequences. It
-can be used for example, for comparing files, and can produce difference
+can be used for example, for comparing files, and can produce different
 information in various formats, including HTML and context and unified
```
Comment (Mariatta) [inline Doc/library/difflib.rst]:
```suggestion
can be used for example, for comparing files, and can produce information
about file differences in various formats, including HTML and context and unified
```

Comment (Mandeepahlawat) [reply Doc/library/difflib.rst]: @Mariatta that definitely sounds much better. I'll go ahead and commit suggestions. Thanks!

**Review by Mariatta — APPROVED** [review]
Thanks!

**Comment by Mariatta** [comment]
Have you sign the CLA?

**Comment by Mandeepahlawat** [comment]
> Have you sign the CLA?

yeah, I signed it after creating this pull request

**Pull request merged**